src/html.sortable.js

```diff
@@ -242,6 +242,12 @@ var _removeItemData = function (items) {
  * @param {Element} destList
  */
 var _listsConnected = function (curList, destList) {
+  var acceptFrom = _data(curList, 'opts').acceptFrom
+  if (acceptFrom !== null) {
+    return acceptFrom !== false && acceptFrom.split(',').filter(
+      sel => sel.length > 0 && _matches(destList, sel)
```

Comment:
One last thing, can you please convert this to an oldschool function:
```
      function (sel) {
        return sel.length > 0 && _matches(destList, sel)
      }
```

Otherwise it throws of the uglify (which would probably be solved with an update) but it also is not supported in IE11